<launch>
    <!--    launch usb cam dependency -->
    <node name="mjpeg_cam" pkg="mjpeg_cam" type="mjpeg_cam" output="screen" >
        <param name="device_name" value="/dev/video0" />
        <param name="width" value="640" />
        <param name="height" value="480" />
        <param name="framerate" value="30" />
        <param name="exposure" value="70" />
        <param name="brightness" value="128" />
        <param name="autoexposure" value="false" />
    </node>
</launch>
